templates/catalog/_partials/miniatures/product.tpl

```diff
@@ -168,6 +168,26 @@
               {/block}
             </div>
 
+            {if $product.add_to_cart_url}
```

Comment:
This button probably shouldn't be displayed if the product requires customization or has attributes.

Reply:
Yes, if product has attributes, display "Choose variant". 🙏